tests/Fixtures/TestBundle/Entity/DummyTableInheritanceNotApiResourceChild.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class DummyTableInheritanceNotApiResourceChild extends DummyTableInheritance
+{
+    /**
+     * Considering this class is not an ApiResource, what should be render is only parent fields.
```

Comment:
Because this class is not tagged as ApiResource, only the parent fields should be in the result.


I don't think that this comment belong here though.

Reply:
Removed.